doc/version_1/062_relatedSystems.md

```diff
@@ -0,0 +1,107 @@
+
+> Impact metrics (Base metric group)
+
+The metrics in this group are Confidentiality, Integrity, and Availability.
+There is also a loosely associated Scope metric.
+The CIA impact metrics are directly handled by [*Technical Impact*](#technical-impact).
+
+Scope is a difficult CVSS metric to categorize.
+The specification describes it as "whether a vulnerability in one vulnerable component impacts resources in components beyond its security scope" [@cvss_v3-1].
+This is a fuzzy concept.
+SSVC better describes this concept by breaking it down into component parts.
+The impact of exploitation of the vulnerable component on other components is covered under [*Mission Impact*](#mission-impact), public and situated [*Well-being Impact*](#well-being-impact), and the stakeholder-specific nature where SSVC is tailored to stakeholder concerns.
+CVSS addresses some definitions of the scope of CVSS as a whole under the Scope metric definition.
+In SSVC, these definitions are in the [Scope](#scope) section.
+
+> Temporal metric groups
+
+The temporal metric group primarily contains the Exploit Code Maturity metric.
+This metric expresses a concept similar to [*Exploitation*](#exploitation).
+The main difference is that [*Exploitation*](#exploitation) is not optional in SSVC and that SSVC accounts for the observation that most vulnerabilities with CVE-IDs do not have public exploit code [@householder2020historical] and are not actively exploited [@guido2011exploit,@jacobs2019exploit].
+
+> Environmental metric group
+
+The environmental metric group allows a consumer of a CVSS base score to change it based on their environment.
+CVSS needs this functionality because the organizations that produce CVSS scores tend to be what SSVC calls **suppliers** and consumers of CVSS scores are what SSVC calls **deployers**.
+These two stakeholder groups have a variety of natural differences, which is why SSVC treats them separately.
+SSVC does not have such customization as a bolt-on optional metric group because SSVC is stakeholder-specific by design.
+
+### EPSS
+
+[EPSS](https://www.first.org/epss/) is an "effort for predicting when software vulnerabilities will be exploited."
+EPSS is currently based on a machine-learning classifier and proprietary IDS alert data from Kenna Security.
+While the group has made an effort to make the ML classifier transparent, ML classifiers are not able to provide an intelligible, human-accessible explanation for their behavior [@spring2019ml].
+The use of proprietary training data makes the system less transparent.
+
+EPSS could be used to inform the [*Exploitation*](#exploitation) decision point.
+Currently, [*Exploitation*](#exploitation) focuses on the observable state of the world at the time of the SSVC decision.
+EPSS is about predicting if a transition will occur from the SSVC state of [*none*](#xploitation) to [*proof of concept*](#exploitation).
```

Comment:
I though EPSS was transition from (none or PoC) to active?

Reply:
Ah, you're right.